pay-api/migrations/versions/cf9a60955b68_bcol_staff_payment.py

```diff
@@ -0,0 +1,35 @@
+"""bcol_staff_payment
+
+Revision ID: cf9a60955b68
+Revises: e8edc889072d
+Create Date: 2020-07-21 08:52:47.896765
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'cf9a60955b68'
+down_revision = 'e8edc889072d'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('corp_type', sa.Column('bcol_staff_fee_code', sa.String(length=20), nullable=True))
+    op.add_column('invoice', sa.Column('dat_number', sa.String(length=50), nullable=True))
+    op.create_index(op.f('ix_invoice_dat_number'), 'invoice', ['dat_number'], unique=False)
```

Comment:
i hope this table doesnt already have a lot of indexes

Reply:
Invoice have some indexes for most of the columns which are searchable.. Yea, we may need to refactor later if there is any performance impacts.

Reply:
okay..